src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Set Up
+sidebar_order: 1
+redirect_from:
+  - /session-replay/setup/
+  - /session-replay/getting-started/
+description: "Get started with Session Replay, TODO."
+---
+
+<Note>
+
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+
+</Note>
+
+To set up the session replay integration, use the links for supported SDKs below. TODO.
+
+## Supported SDKs
+
+- JavaScript
```

Comment:
We should add the plain Browser SDK (@sentry/browser) here for users who don't rely on a framework